src/docs/product/performance/filters-display.mdx

```diff
@@ -59,6 +65,20 @@ You can compare graphs side by side to find any relevant correlations. For examp
 - [p95 Duration](/product/performance/metrics/#p95-threshold)
 - [p99 Duration](/product/performance/metrics/#p99-threshold)
 
+## Widgets
+
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+Widgets offer visualizations that you can change to best match your workflow. In the first row, there are time series and distribution graphs of common metrics for the selected project type (frontend or backend), such as User Misery or Failure Rate. You can compare these side-by-side to find any correlations. 
```

Comment:
Should we mention fe, be or mobile?

Reply:
Yeah we can, maybe just mention there are different types of widgets to compare since there are different view?